polkadot-wiki/sidebars.js

```diff
@@ -421,6 +421,8 @@ module.exports = {
                   },
                   items: [
                     "learn/learn-agile-coretime",
+                    "learn/learn-guides-coretime-marketplaces",
+                    "learn/learn-guides-coretime-parachains",
```

Comment:
Maybe this belongs in Build? I am not entirely opposed to it being here, but it may fit better to discuss cores in Build and then link immediately in this guide. WDYT?